src/openrct2-ui/windows/About.cpp

```diff
@@ -224,31 +226,41 @@ static void window_about_openrct2_common_paint(rct_window * w, rct_drawpixelinfo
 
 static void window_about_openrct2_paint(rct_window *w, rct_drawpixelinfo *dpi)
 {
-    window_draw_widgets(w, dpi);
     window_about_openrct2_common_paint(w, dpi);
 
     sint32 x, y, width;
     rct_size16 logoSize;
 
+    sint32 lineHeight = font_get_line_height(gCurrentFontSpriteBase);
+
     x = w->x + (w->width / 2);
-    y = w->y + w->widgets[WIDX_PAGE_BACKGROUND].top + 10;
+    y = w->y + w->widgets[WIDX_PAGE_BACKGROUND].top + lineHeight;
     width = w->width - 20;
-    
-    utf8 buffer[256];
-    utf8 *ch = buffer;
-    openrct2_write_full_version_info(ch, sizeof(buffer) - (ch - buffer));
 
-    y += gfx_draw_string_centred_wrapped(dpi, nullptr, x, y, width, STR_ABOUT_OPENRCT2_DESCRIPTION, w->colours[2]) + 10;
+    y += gfx_draw_string_centred_wrapped(dpi, nullptr, x, y, width, STR_ABOUT_OPENRCT2_DESCRIPTION, w->colours[2]) + lineHeight;
 
     logoSize = gfx_get_sprite_size(SPR_G2_LOGO);
     gfx_draw_sprite(dpi, SPR_G2_LOGO, x - (logoSize.width / 2), y, 0);
-    y += logoSize.height + 20;
-
-    y += gfx_draw_string_centred_wrapped(dpi, nullptr, x, y, width, STR_ABOUT_OPENRCT2_DESCRIPTION_2, w->colours[2]) + 15;
-    gfx_draw_string_centred_wrapped(dpi, nullptr, x, y, width, STR_ABOUT_OPENRCT2_DESCRIPTION_3, w->colours[2]);
+    y += logoSize.height + lineHeight * 2;
+
+    // About OpenRCT2 text
+    y += gfx_draw_string_centred_wrapped(dpi, nullptr, x, y, width, STR_ABOUT_OPENRCT2_DESCRIPTION_2, w->colours[2]) + lineHeight + 5;
+
+#ifndef NO_TTF
```

Comment:
I doubt this needs to be ifdef'ed - gUseTrueTypeFont is always defined.

Reply:
Ah, you're right. I was being overly cautious — it looks like it is always defined. I'll adjust the code, and clarify the comment.